interface/reports/collections_report.php

```diff
@@ -529,24 +558,7 @@ function checkAll(checked) {
                         <?php echo xlt('Payor'); ?>:
                         </td>
                         <td>
-                        <?php  # added dropdown for payors (TLH)
-                               $insurancei = getInsuranceProviders();
-                               echo "   <select name='form_payer_id' class='form-control'>\n";
-                               echo "    <option value='0'>-- " . xlt('All') . " --</option>\n";
-                        foreach ($insurancei as $iid => $iname) {
-                            echo "<option value='" . attr($iid) . "'";
-                            if (!empty($_POST['form_payer_id']) && ($iid == $_POST['form_payer_id'])) {
-                                echo " selected";
-                            }
-
-                            echo ">" . text($iname) . "</option>\n";
-                            if (!empty($_POST['form_payer_id']) && ($iid == $_POST['form_payer_id'])) {
-                                $ins_co_name = $iname;
-                            }
-                        }
-
-                               echo "   </select>\n";
-                        ?>
+                            insuranceSelect();
```

Comment:
hi @vsdevjassim , would move the php closing tag down below this so the insurance drop down is functioning